### ISSUE TITLE
Some plugins can no longer be disabled

### ISSUE THREAD

**Issue by mjmaax** [getgrav/grav#2509]
Some plugins stay active even after setting enable to false in the yaml. I cleared the cache but the problem remains. That's the case with langswitcher for example.

**Comment by rhukster** [comment]
Admin/Form/Login/Email cannot be disabled when in the admin.  Other than that there's no rules that I know of.  Langswitcher is primarily a twig include, so maybe there's no check if the plugin is enabled in your Twig?

**Comment by mjmaax** [comment]
Thank you for the reply.

I thought that Grav itself would not load a plugin if 'enabled' was set to false in the yaml. Would make sens from a security point of view... or if a plugin is broken...
Or is it only an attribute that the plugins are free to use or not?

**Comment by rhukster** [comment]
If a plugin is disabled, it should not be loaded at all.  No events are registered by the plugin, so no logic should run for that plugin.  The only thing I can think of is if you have an override for a twig template ( that would normally be added to twig paths, but not when it's disabled) in your theme is available, and no check is in place before trying to use that twig.  If that makes sense?

BTW, I just tested with the multilang skeleton (https://github.com/getgrav/grav-skeleton-multilang-site) with admin.  I disabled the `langswitcher` plugin and it didn't show up on the site.  So it's definitely working.  Could be something related to how your theme works?
